cmd/main_test.go

```diff
@@ -267,14 +93,14 @@ var _ = BeforeSuite(func() {
 	user2 := "user2@konflux.dev"
 	namespaceNames := []string{"test-tenant", "test-tenant-2", "test-tenant-3"}
 	for _, name := range namespaceNames {
-		_, err := createNamespace(k8sClient, name)
+		_, err := utils.CreateNamespace(k8sClient, name)
 		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error while creating the namespace %s: %v", name, err))
 	}
-	createRole(k8sClient, "test-tenant", "namespace-access", []string{"create", "list", "watch", "delete"})
-	createRole(k8sClient, "test-tenant-2", "namespace-access-2", []string{"create", "list", "watch", "delete"})
-	createRoleBinding(k8sClient, "namespace-access-user-binding", "test-tenant", user1, "namespace-access")
-	createRoleBinding(k8sClient, "namespace-access-user-binding-2", "test-tenant", user2, "namespace-access")
-	createRoleBinding(k8sClient, "namespace-access-user-binding-3", "test-tenant-2", user2, "namespace-access-2")
+	utils.CreateRole(k8sClient, "test-tenant", "namespace-access", []string{"create", "list", "watch", "delete"})
+	utils.CreateRole(k8sClient, "test-tenant-2", "namespace-access-2", []string{"create", "list", "watch", "delete"})
+	utils.CreateRoleBinding(k8sClient, "namespace-access-user-binding", "test-tenant", user1, "namespace-access")
+	utils.CreateRoleBinding(k8sClient, "namespace-access-user-binding-2", "test-tenant", user2, "namespace-access")
+	utils.CreateRoleBinding(k8sClient, "namespace-access-user-binding-3", "test-tenant-2", user2, "namespace-access-2")
```

Comment:
Can we import the different functions from utils and keep the code as-is instead of changing all usages?

Reply:
moved these functions back to main_test.go